macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -32,7 +31,9 @@
                 {% if dimensions %}
                     and dimension = {{ elementary.quote(elementary.join_list(dimensions, '; ')) }}
                 {% endif %}
-
+                {% if metric_properties %}
```

Comment:
I think this should be at the top and not conditional (without the `if`).

Reply:
done